pkg/capacity/capacity.go

```diff
@@ -231,6 +231,7 @@ func (c *Controller) Run(ctx context.Context, threadiness int) {
 	klog.Info("Starting Capacity Controller")
 	defer c.queue.ShutDown()
 
+	go c.topologyInformer.RunWorker(ctx)
```

Comment:
I still think this belongs into csi-provisioner.go. We should add a comment to `NewCentralCapacityController` that "all informers are expected to be started by the caller".

Reply:
Node topology worker only works for Capacity controller. If something goes wrong before CapacityController Run, I think we don't even need to start this worker. Therefore, why not put it into controller run function :)

Reply:
Because it is consistent conceptually ("*all* informers are expected to be started by the caller"- we can even add "started *and stopped*").

If something goes wrong in csi-provisioner.go, the binary will exit. It'll make no difference how many goroutines get killed by that and the overhead is also irrelevant.